### PR TITLE
Patch Ajax request to include headers per heroku api mandate

### PR DESCRIPTION
**Pull request by quadrophobiac**
per @pezholio’s link https://api.heroku.com/apps/hello
this is intended to fix issue #10

### CODE REVIEW

**Comment by quadrophobiac** [comment]
So I figure a breaking issue like this warrants testing. However the existing test suite does not fail (see any one of the other PRs currently open) in light of new API changes.
Testing this locally is also fraught. Any `GET` to `'/'` results in a redirect to the live site (which is the expected behaviour, as exercised in [bothan_deploy_spec](https://github.com/theodi/bothan-deploy/blob/master/spec/bothan_deploy/bothan_deploy_spec.rb#L5-L8) and [spec_helper](https://github.com/theodi/bothan-deploy/blob/master/spec/spec_helper.rb#L37-L40). 
The way around that is to disable some heroku `.env` variables, and then there is CORS to be worked around. 
I tried to use Heroku review apps but then a whole can of worms opened there.
So in short - if this looks like a resolution to issue #10 can it be merged in?

**Comment by quadrophobiac** [comment]
got review apps working - this is not good to merge

**Comment by quadrophobiac** [comment]
EDIT EDIT what I said RE review apps - I'm not sure that heroku review apps can be used to test this either because they too redirect to https://deploy.bothan.io/ rather than their actual URL.
@pezholio, @Floppy  I'm out of knowledge on the best practice here - what's the best way to test that this is operational in the same way it will be live.
I get the sense that the redirect behaviour is a function of [heroku-bouncer](https://github.com/heroku/heroku-bouncer), but I'm not certain. Looking for an expedient resolution to this if it exists but I don't know where to begin in terms of amending the redirect behaviour that's causing the testing issue

**Comment by quadrophobiac** [comment]
Have tested this manually like so

* remove HEROKU_OAUTH_ID and HEROKU_OAUTH_SECRET to disable heroku bouncer middleware
* run app via `RACK_ENV=test bundle exec rackup config.ru`
* navigate to localhost and enter text into the `Title` text field. Existing apps like `demo` should return a 200 code, a string of characters that has no matching heroku app should return a 400

**Comment by Floppy** [comment]
Live's broken anyway so let's give it its final test there :)
